flink-table/flink-sql-parser/src/main/codegen/templates/Parser.jj

```diff
@@ -0,0 +1,8469 @@
+/*
```

Comment:
Keeping a copy of Parser.jj is very risky. Have you considered `Extending the Parser`[1]?

[1] https://calcite.apache.org/docs/adapter.html#extending-the-parser

Reply:
it is already a part of Flink as it was added somewhere in May with another commit